apply.sh

```diff
@@ -49,20 +55,29 @@ fi
 cd /nix/home/darwin-config
 nix-shell -p git --run "git fetch $CONFIG_REPO $CONFIG_BRANCH && git checkout FETCH_HEAD"
 
+nix --extra-experimental-features 'nix-command flakes' profile install nixpkgs#git
+
+config="/nix/home/darwin-config"
+nixpkgs=$(nix --extra-experimental-features 'nix-command flakes' eval --raw "$config"#inputs.nixpkgs)
+darwin=$(nix --extra-experimental-features 'nix-command flakes' eval --raw "$config"#inputs.darwin)
+export NIX_PATH=darwin-config="$config/$CONFIG_TARGET":nixpkgs=$nixpkgs:darwin=$darwin
```

Comment:
I wonder if we could just ... delete these... :)

Reply:
(...and other refs to NIX_PATH here.)

Reply:
Yep, all done.